Rubberduck.Deployment/InnoSetup/Rubberduck.Installer.Build.iss

```diff
@@ -70,21 +70,10 @@ Uninstallable=ShouldCreateUninstaller()
 CreateUninstallRegKey=ShouldCreateUninstaller()
 
 ; should be at least a 55 x 55 bitmap
-WizardSmallImageFile={#GraphicsDir}Rubberduck.Duck.Small.55x55.bmp, \
-                     {#GraphicsDir}Rubberduck.Duck.Small.64x68.bmp, \
-                     {#GraphicsDir}Rubberduck.Duck.Small.83x80.bmp, \
-                     {#GraphicsDir}Rubberduck.Duck.Small.92x97.bmp, \
-                     {#GraphicsDir}Rubberduck.Duck.Small.110x106.bmp, \
-                     {#GraphicsDir}Rubberduck.Duck.Small.119x123.bmp, \
-                     {#GraphicsDir}Rubberduck.Duck.Small.138x140.bmp
+WizardSmallImageFile={#GraphicsDir}Duck.Small\*.bmp
```

Comment:
this should be `{#GraphicsDir}Rubberduck.Duck.Small*.bmp`. Same change applies for the `WizardImageFile`

Reply:
Oh ok cool, I was using subfolders but I guess that's redundant. Will change.